Cookbook/Technical-Documents/ApplicationDistribution.md

```diff
@@ -0,0 +1,24 @@
+# Application Distribution
+This document is for any Babylon iOS developers who wish to distribute our applications, or for any partners who wish to gain access to our applications.
```

Comment:
```suggestion
This document is for Babylon iOS developers for distribution of our applications internally, to other Babylon employees, or externally, to our partners.
 ```

Reply:
I think this reads better with your suggestion, keeping the `who wish` part in it - what do you think?

Reply:
sure. That is ok